### PR TITLE
testsuite: add test for RequireAbsolutePath option

### PR DESCRIPTION
**Pull request by mhabrnal**
The option is placed in '/etc/abrt/plugins/python.conf' and
the option is placed in '/etc/abrt/plugins/python3.conf'.

Signed-off-by: Matej Habrnal <mhabrnal@redhat.com>

### CODE REVIEW

File: tests/runtests/python-addon/runtest.sh [hunk 1]
```diff
@@ -57,6 +59,37 @@ rlJournalStart
...
+1/0
+EOF
+        rlRun "chmod +x $CRASH_PY" 0
+
+        rlRun "augtool set /files${AASPD_CONF}/ProcessUnpackaged yes" 0
+        rlRun "augtool set /files${PYTHON_CONF}/RequireAbsolutePath yes" 0
+
+        prepare
+        rlRun "./$CRASH_PY" 1
+        rlAssertNotExists $crash_PATH
```
Comment (jfilak) [inline tests/runtests/python-addon/runtest.sh]: This asserts test if `$crash_PATH` does not exist but if the crash does not exist, `$crash_PATH` is empty. What's worse, `$crash_PATH` is populated by the previous test which also removes the related problem, so the variable is not empty and the path will never exist.

File: tests/runtests/python-addon/runtest.sh [hunk 2]
```diff
@@ -57,6 +59,37 @@ rlJournalStart
...
+
+        rlRun "augtool set /files${PYTHON_CONF}/RequireAbsolutePath no" 0
+
+        prepare
+        rlRun "./$CRASH_PY" 1
+        wait_for_hooks
+        get_crash_path
+        check_dump_dir_attributes $crash_PATH
```
Comment (jfilak) [inline tests/runtests/python-addon/runtest.sh]: What if abrt python handler causes python to crash?

Comment (mhabrnal) [inline tests/runtests/python-addon/runtest.sh]: I've added a check if ABRT caught the right crash.

**Comment by mhabrnal** [comment]
Updated

**Comment by jfilak** [comment]
Thank you! Good job.

**Pull request merged**